src/main/java/com/onaio/steps/handler/strategies/survey/TakeSurveyForHouseholdStrategy.java

```diff
@@ -69,6 +69,8 @@ public void handleResult(ODKSavedForm savedForm){
             household.setStatus(InterviewStatus.INCOMPLETE);
 
         household.setOdkFormId(savedForm.getId());
+        household.setOdkJrFormId(savedForm.getJrFormId());
+        household.setOdkJrFormTitle(savedForm.getDisplayName());
```

Comment:
Add a test to assert these values in the respective test file https://github.com/onaio/steps-app/blob/65114592848ccea3d40bbf8b07d09dc7acb93562/src/test/java/com/onaio/steps/handler/strategies/survey/TakeSurveyForHouseholdStrategyTest.java#L77

Reply:
assertion added.